src/CoreWCF.AzureQueueStorage/src/CoreWCF/Channels/AzureQueueStorageConnectionSettings.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Storage.Queues;
+
+namespace CoreWCF.Channels
+{
+    public class AzureQueueStorageConnectionSettings
+    {
+        public static QueueClient GetQueueClientFromConnectionString(
+            string connectionString,
```

Comment:
Does this class need to be public?

Reply:
So this can be made internal, but then the unit tests wont be able access them, so i have left it as public

Reply:
Testability isn't a good reason to make something public. It being public means it needs to be documented, and it must stay around indefinitely so no refactoring is possible. This class is stand alone so you could just source include it into the test project. Also I just realized, this class could be static.

Reply:
yeah thats a good point, i looked at this class and realized that there is no need for this class anymore since the I have separate classes for the Queueclient. so, i can just delete this file.